### PR TITLE
adding table of contents to the README.md doc

### PR DESCRIPTION
**Pull request by Samuellyworld**
- [x] better readme content with table of content.
- [x] helps to onboard new developers

image

### CODE REVIEW

**Comment by Samuellyworld** [comment]
@armaniferrante @steveluscher @bhgames @johnrees @tomlinton kindly review.🥑

File: README.md [hunk 1]
```diff
@@ -30,15 +47,15 @@ and add it to your local chrome profile, using developer mode. See the video bel
 
 https://user-images.githubusercontent.com/101902546/173857300-fc139113-0af5-46fc-baad-236a2ebf63f1.m4p
 
-### 0. Pull the code
+###  Pull the code
 
-```
+```bash
 git clone git@github.com:coral-xyz/backpack.git
 git submodule init
```
Comment (tomlinton) [inline README.md:54]: Could you add the suggestion from https://github.com/coral-xyz/backpack/issues/1010 so we can close that also?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,23 @@
   </p>
 </div>
 
+# Table of Content:
+
+- [Note](#note)
```
[the remark this answers is not in the document]
Comment (tomlinton) [reply README.md:20]: Agreed, lets move this above and remove it from the TOC.

**Review by Samuellyworld — COMMENTED** [review]
added notes in the sub header instead of TOC, kindly review.

File: README.md [hunk 3]
```diff
@@ -48,33 +62,33 @@ Go to chrome://flags/#allow-insecure-localhost and enable the toggle, then resta
 
 You can also optionally rename `.env.example` to `.env` and set your own variables.
 
-### 2. Install dependencies
+###  Install dependencies
```
Comment (tomlinton) [inline README.md:65]:
```suggestion
### Install dependencies
```

File: README.md [hunk 4]
```diff
@@ -48,33 +62,33 @@ Go to chrome://flags/#allow-insecure-localhost and enable the toggle, then resta
...
 
-```
+```bash
 yarn install
 ```
 
-### 3. Build all packages for production
+###  Build all packages for production
```
Comment (tomlinton) [inline README.md:71]:
```suggestion
### Build all packages for production
```

File: README.md [hunk 5]
```diff
@@ -48,33 +62,33 @@ Go to chrome://flags/#allow-insecure-localhost and enable the toggle, then resta
...
 
-```
+```bash
 yarn build
 ```
 
-### 4. Start packages for development
+###  Start packages for development
```
Comment (tomlinton) [inline README.md:77]:
```suggestion
### Start packages for development
```

**Pull request merged**